.github/workflows/deployment-actions.yml

```diff
@@ -1,3 +1,8 @@
+# The purpose of this workflow file is to 
+# deploy the app and upload the
+# release asset for the four platforms 
```

Comment:
Good Job!  Just a slight typo that you might have missed out on! 

```suggestion
# release assets for the four platforms 
```

Reply:
I will make the necessary amendment, thank you so much!